### DISCUSSION TITLE
Homekit Integration

### DISCUSSION THREAD

**Discussion by xAIBuilder** [briis/unifiprotect#311] [Q&A]
I have this installed and configured with the G4 doorbell showing on a lovelace dashboard. Is there a way to also have it show up in Homekit integration as well? I have other items currently showing up such as sonos and a garage door (cover) but I am not sure how to add this. The Homekit integration is configured in bridge mode. I selected camera from the domains on the homekit integration, the g4 camera is available, and on the camera configuration I have tried both with and without the native h.264 support but it does not show up in homekit.

**Comment by sose5000** [comment]
Would also love to know.  Moving away from a handful of cameras that are in HA and HK.  Unifi cameras do not get passed to HK.

**Reply by bdraco** [reply]
https://www.home-assistant.io/integrations/homekit/#accessory-mode

Please read the section `To quickly add all accessory mode entities in the UI:`

**Reply by sose5000** [reply]
thank you!
